### PR TITLE
Setting character limits for text fields

### LINKED ISSUE #476: Add character limits for question 6 follow-up text fields
Employer Name - 500 characters
Address Line 1 - 500 characters
Address Line 2 - 500 characters
City - 100 characters
Zip - 10 characters
Total Hours - 6 characters
Gross Earnings - 11 characters
More Details - 2000 characters

### PR DESCRIPTION
**Pull request by sharonwarner**
- Adds character limits for the 8 text fields in the employer section of the app

===

Resolves #476 

- [ ] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: src/client/components/EmployersQuestions/index.js [hunk 1]
```diff
@@ -182,10 +184,12 @@ function EmployerQuestions(props) {
       {renderTextInput("totalHoursWorked", {
         className: "col-md-2",
         pattern: "\\d+[.]?\\d*",
+        maxLength: 3,
```
Comment (idealisms) [inline src/client/components/EmployersQuestions/index.js:187]: Do we even need to allow fractional hours? Should we update the pattern to just \d+?

Comment (sharonwarner) [reply src/client/components/EmployersQuestions/index.js:187]: Oh good point! The max should probably be 6 characters if we're allowing decimals. @nguyen-tina do you have a preference here? I'll also check the DB to see if anybody has submitted decimals yet.

Comment (nguyen-tina) [reply src/client/components/EmployersQuestions/index.js:187]: Yes, we should allow for decimals and extra characters to get fractional hours! I agree with changing to max 6 characters

Comment (sharonwarner) [reply src/client/components/EmployersQuestions/index.js:187]: Sounds good! Updating now!

**Pull request merged**